devApps/AdalAndroidTestApp/MainActivity.cs

```diff
@@ -32,19 +32,25 @@
 using Android.Widget;
 using Android.OS;
 using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using System.Text;
 
 namespace AdalAndroidTestApp
 {
     [Activity(Label = "AdalAndroidTestApp", MainLauncher = true, Icon = "@drawable/icon")]
     public class MainActivity : Activity
     {
-        private const string clientId = "d3590ed6-52b3-4102-aeff-aad2292ab01c";
```

Comment:
Where is this clientId from? asking because depending on the machine, we'll need to update the redirectUri.